lang/csharp/src/apache/main/Specific/ObjectCreator.cs

```diff
@@ -142,7 +142,14 @@ private Type FindType(string name,bool throwError)
                     if (assembly.FullName.StartsWith("MonoDevelop.NUnit"))
                         continue;
 
-                    types = assembly.GetTypes();
+                    try
+                    {
+                        types = assembly.GetTypes();
+                    }
+                    catch
```

Comment:
Would it be worthwhile, from a debugging perspective, to log the assembly + exception so people may know why an assembly was not found?

Reply:
not a bad idea, and the avro library makes use of log4net, but on the other hand i suspect not many people are going to make use of it.